engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/canonicalgraph/ProcessNodesRewriter.scala

```diff
@@ -144,7 +144,13 @@ trait ExpressionRewriter {
       case n: Source =>
         n.copy(
           ref = n.ref.copy(parameters = rewriteParameters(n.ref.parameters)))
-      case _: BranchEndData | _: Split | _: SubprocessInputDefinition | _: SubprocessOutputDefinition | _: SubprocessOutput => data
+      case n: SubprocessOutputDefinition =>
+        n.copy(
```

Comment:
Ugly formatting.. Better will be in one line?

Reply:
Ugliness is a matter of tase probably :)
but back to this match clasue - I've kept the existing code formatting, for sake of style consistency